docs/landing/layouts/partials/features.html

```diff
@@ -7,8 +7,13 @@ <h2>Features</h2>
     intermediate Map instance.</dd>
   <dt>MongoDB Driver</dt>
   <dd>An updated Java driver that includes the legacy API as well as a new generic MongoCollection interface that complies with a new cross-driver CRUD specification.</dd>
-  <dt>MongoDB Async Driver</dt>
-  <dd>A callback-based asynchronous driver.  Note that this driver is now deprecated in favor of the
-      <a href="http://mongodb.github.io/mongo-java-driver-reactivestreams/">Reactive Streams Java Driver</a>
+  <dt>MongoDB Reactive Streams Driver</dt>
```

Comment:
Nice.  But maybe also say there that RS/Scala are documented here starting in 4.0.

Reply:
Done